podcasts/PlaybackManager.swift

```diff
@@ -2055,6 +2055,18 @@ class PlaybackManager: ServerPlaybackDelegate {
             // the current episode we were playing has downloaded, switch to playing the downloaded version
             let currentlyPlaying = playing()
             recordPlaybackPosition(sendToServerImmediately: false, fireNotifications: true)
+
+            let episodeIsChanging = refreshedEpisode.uuid != currentEpisode()?.uuid
+
+            if FeatureFlag.doNotSwitchToDownloadedFile.enabled,
```

Comment:
can we wrap this into a method in case we need to re-use it? Also would be great to add few logs including the param values. Wdyt?

Reply:
Do you want to log the episode uuid?

Reply:
Maybe the episode title and the reasons of why is switching. In case we get some reports of pop sound and we find on log it didn't switch